config/proto/targets.proto

```diff
@@ -0,0 +1,44 @@
+/*
+  This proto defines the config for a Target. 
+  Targets are used by Hermes and Cloudprober to identify storage systems to monitor.
+*/
+syntax = "proto3";
+
+package hermes;
+
+option go_package = "github.com/googleinterns/step224-2020/targets/proto";
+
+// Definition of a Target, a storage system to be probed by a specific probe.
+message TargetsDef {
+  enum TargetSystem {
+    GOOGLE_CLOUD_STORAGE = 0;
+    CEPH = 1;
+  }
+
+  enum ConnectionType {
+     HTTP = 0;
+     HTTPS = 1;
+  }
+
+  TargetSystem target_system = 1;
+  ConnectionType connection_type = 2;
+  ConnectionDetails connection_details = 3;
+  int64 total_space_allocated = 4; // in MBs.
+}
+
+// Necessary when connecting to an API (e.g. Ceph).
+// Unclear yet if we need this for GCS, as we will use the Go client library and service account credentials.
```

Comment:
If it is unclear this will be needed for GCS then only make it required when the connection type is Ceph, so just make a comment on it. And just mention that this functionality will be handled in the Go client library/binary for GCS. If that changes, then just come back here when it does and update the comment.

Reply:
```suggestion
// Required for Ceph.
// This functionality will be handled in the Go client library for GCS.
```